packages/static-wado-scp/lib/dicomWebScpConfig.js

```diff
@@ -9,8 +9,8 @@ const { dicomWebScpConfig } = ConfigPoint.register({
     configBase: staticWadoConfig,
     isStudyData: true,
     isGroup: true,
-    maximumInlinePrivateLength: 128 * 1024 + 2,
-    maximumInlinePublicLength: 64,
+    maximumInlinePrivateLength: 64,
```

Comment:
Noticed this going through things - the maximum private length should be very short, while the maximum public one should be fairly long.